modules/persistent-storage-csi-ebs-operator-install.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-install_{context}"]
+= Installing the AWS Elastic Block Store CSI Operator
+
+The AWS Elastic Block Store (EBS) Container Storage Interface (CSI) Operator enables the replacement of the existing AWS EBS in-tree storage plug-in.
+
+When you install the AWS EBS CSI Operator, {product-title} provides the CSI driver that allows you to use CSI volumes with the `PersistentVolumeClaims`, `PersistentVolumes`, and `StorageClasses` API objects.
+
+The AWS EBS CSI Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the AWS EBS CSI driver in your cluster.
+
+.Prerequisites
+* Access to the `oc` command-line interface (CLI) or the {product-title} web console.
```

Comment:
I don't see the procedure to install though the CLI. Do we need to add it? Or should we remove the `oc` prerequisite?

Reply:
Good catch! We removed the `create project` CLI instructions, so we can remove the `oc` prereq here.